homeassistant/components/barry/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "domain": "barry",
+  "name": "Barry",
+  "documentation": "https://www.home-assistant.io/integrations/barry/",
+  "dependencies": [],
+  "codeowners": ["@YashYKhatri", "@ThomasBLarsen"],
+  "requirements": ["pybarry==0.0.4"],
+  "quality_scale": "silver",
```

Comment:
Are you sure you match all items of this list? https://developers.home-assistant.io/docs/integration_quality_scale_index/#silver-%F0%9F%A5%88.

Reply:
Not at moment. I changed the scale to "internal" and will improve the quality in the next version.

Reply:
>Integrations which are part of Home Assistant are not rated but marked as internal.

You can not use `internal` for this integration. Best would be to remove the quality scale from this PR for now.